src/applications/static-pages/i18Select/I18Select.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+import recordEvent from 'platform/monitoring/record-event';
+
+const i18Content = {
+  en: {
+    label: 'English',
+    suffix: '/',
+  },
+  es: {
+    onThisPage: 'En esta página',
+    label: 'Español',
+    suffix: '-esp/',
+  },
+  tag: {
+    suffix: '-tag/',
+    label: 'Tagalog',
+    onThisPage: 'Tagalog On this page',
+  },
+};
+const langsToLinkSuffixes = {
+  es: '-esp/',
+  tag: '-tag/',
+};
+const I18Select = ({ baseUrls }) => {
+  const [lang, setLang] = useState('en');
+  useEffect(() => {
+    const contentDiv = document?.getElementById('content');
+
+    setLang('en');
+    if (contentDiv) {
+      contentDiv.setAttribute('lang', 'en');
+    }
+    for (const [langCode, suffix] of Object.entries(langsToLinkSuffixes)) {
+      if (document?.location.href.endsWith(suffix)) {
+        setLang(langCode);
+        if (contentDiv) {
+          contentDiv.setAttribute('lang', langCode);
+        }
+      }
+    }
+  }, []);
+
+  useEffect(
+    () => {
+      if (lang && lang !== 'en') {
+        const onThisPageEl = document?.getElementById('on-this-page');
+        onThisPageEl.innerText = i18Content[lang].onThisPage;
+      }
+    },
+    [lang],
+  );
+
+  const handleLinkClick = keyValue => {
+    recordEvent({
+      event: 'nav-covid-link-click',
+      faqText: undefined,
+      faqSection: undefined,
+    });
+    const translatedPage = baseUrls[keyValue];
+    if (translatedPage) {
+      document.location.pathname = translatedPage;
+    }
+  };
+
+  return (
+    <div className="vads-u-display--inline-block vads-u-margin-top--4 vads-u-margin-bottom--3 vads-u-border--0 vads-u-border-bottom--1px vads-u-border-style--solid vads-u-border-color--gray">
+      <span>
+        {Object.entries(i18Content).map(([languageCode, languageConfig], i) => {
+          return (
+            <span key={i}>
+              <a
```

Comment:
we need it b/c of the whole "sibling elements need one parent" jsx rule, since the delimiter " | " is a sibling to the link.  if you have an alternate suggestion i'm all ears 👂

Reply:
Oh I see, makes sense